src/selectors/modular-pipelines.js

```diff
@@ -44,7 +44,12 @@ export const searchTree = (
       // if the child node is a leaf, simply search the leaf's name
       // and add to the search result if there is a match.
       const found = searchString(childNode.data.name, searchValue);
-      if (found) {
+      const foundOpposite = searchString(
```

Comment:
[Nit]  I feel opposite terminology not fitting but at the same time, I cannot come up with a suitable name here. @tynandebold any suggestions ?

Reply:
I'm fine with it.